packages/gitgraph-core/src/__tests__/gitgraph.clear.test.ts

```diff
@@ -37,15 +37,9 @@ describe("Gitgraph.clear", () => {
     const core = new GitgraphCore();
     const gitgraph = core.getUserApi();
 
-    gitgraph
-      .branch("dev")
-      .commit("one")
-      .commit("two");
+    gitgraph.branch("dev").commit("one").commit("two");
```

Comment:
Aha, new Prettier version 😆

Reply:
Haha yeah. I noticed that after working through this PR, but it's the default unfortunately